e2e_test/js-tests/test_viem_tx.mjs

```diff
@@ -271,27 +309,38 @@ describe("viem send tx", () => {
 		}
 	}).timeout(10_000);
 
-	it("send fee currency tx with just high enough gas price", async () => {
+	it("send fee currency tx with just high enough gas price mine", async () => {
+		const rate = await getRate(process.env.FEE_CURRENCY);
 		const block = await publicClient.getBlock({});
-		// Actually, the base fee will be a bit lower next block, since our blocks
-		// are always mostly empty. But the difference will be much less than the
-		// exchange rate of 2, so the test will still check that we take the fee
-		// currency into account everywhere.
-		const maxFeePerGas = block.baseFeePerGas / 2n;
+		const maxFeePerGas = rate.toFeeCurrency(block.baseFeePerGas)+2n;
```

Comment:
In case of increasing base fees, this might not be enough to get the tx into a block.

Reply:
What do you suggest? Right now this seems to be working reliably, but could get flaky pretty easily.

Reply:
Hey @karlb, I've updated the test to address your suggestion.